hub/powertoys/cmd-not-found.md

```diff
@@ -0,0 +1,36 @@
+---
+title: PowerToys Command Not Found for Windows
+description: A PowerShell 7 module that detects an error thrown by a command and suggests a relevant WinGet package to install, if available.
+ms.date: 02/01/2024
+ms.topic: article
+no-loc: [PowerToys, Windows, Command Not Found, Win]
+---
+
+# Command Not Found utility
+
+A PowerShell 7 module that detects an error thrown by a command and suggests a relevant WinGet package to install, if available.
+
+![AlwaysOnTop screenshot.](../images/pt-cmd-not-found.png)
+
+## Requirements
+ - [PowerShell 7](https://learn.microsoft.com/en-us/PowerShell/scripting/install/installing-PowerShell-on-windows?view=PowerShell-7.4)
+ - [PowerShell Microsoft.WinGet.Client module](https://www.powershellgallery.com/packages/Microsoft.WinGet.Client)
+
+## Install the module
+
+To install the Command Not Found module, navigate to PowerToys settings Command Not Found page and press Install button. On module install, PowerShell 7 experimental features needed for the module to function will be enabled:
```

Comment:
```suggestion
To install the Command Not Found module, go to PowerToys settings Command Not Found page and select **Install**. After installation, PowerShell 7 experimental features needed for the module to function will be enabled:
```

Reply:
I just left a suggestion for this same block. Can you combine the suggestions into an updated paragraph, @stefansjfw?

Reply:
Haha, we're thinking the same thing! :)